paasta_tools/kubernetes_tools.py

```diff
@@ -1000,6 +1004,16 @@ def get_secret_volume_name(self, secret_volume: SecretVolume) -> str:
             "secret--{name}".format(name=secret_volume["secret_name"]), length_limit=63
         )
 
+    def get_projected_volume_name(self, projected_volume: ProjectedVolume) -> str:
```

Comment:
as written, this means that projected_volumes will only support `serviceAccountToken` as a source, but that's probably fine since we can mount secrets/configmaps as files in other ways

Reply:
yes, I wanted to avoid having multiple ways of doing the same thing (at least in a first iteration).